worker/uniter/util_test.go

```diff
@@ -414,12 +416,14 @@ type createUniter struct {
 func (s createUniter) step(c *gc.C, ctx *context) {
 	step(c, ctx, ensureStateWorker{})
 	step(c, ctx, createApplicationAndUnit{})
+	//step(c, ctx, createApplicationAndUnit{applicationName: "wordpress"})
```

Comment:
I'll remove this

Reply:
??

Reply:
darn gremlins, i remember removing this.  ty, will check that the change got pushed up.